### PR TITLE
Dom library

### PR DESCRIPTION
**Pull request by trekab**
## Project: Library:

In this project we:
- Added the Library HTML.
- Added JS functionality for adding and removing books from the library.
- Added ability for the user to toggle the read status of any book in the library.

### CODE REVIEW

File: js/script.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+function render(bookArray) {
+  bookList.innerHTML = '';
+  bookArray.forEach((book, index) => {
+    // eslint-disable-next-line no-use-before-define
+    bookList.appendChild(createCard(book, index));
+  });
+}
+
+function createCard(book, id) {
```
Comment (bellom) [inline js/script.js:40]: Let's separate the DOM from application logic, you can create another file for DOM.

image

https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59623451/

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi Team:clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 
 
**Kindly check out my comments below;**

**Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....**

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Review by shloch — APPROVED** [review]
## Good to go
Logic and DOM manipulation are seprate
App is beautiful
Screen Shot 2020-05-08 at 11 46 49 AM

**Comment by trekab** [comment]
Thank you @bellom and @shloch for the timely code reviews. :smile:

**Pull request merged**